README.md

```diff
@@ -108,19 +104,52 @@ Before you begin, ensure you have the following software installed on your syste
 
     If you encounter an error during this step, try running the production Docker Compose file located in the repository under `Docker-compose.yml`. After a successful build, you can retry step 4.
 
-### Setting up Scopes for Anonymous User in Gateway UI
-
-In the Gateway UI, you need to configure scopes for the anonymous user. Follow these steps to set it up:
-
-1.  Go to the settings in the Gateway UI.
-2.  Navigate to the 'Security Groups' tab.
-3.  Locate and select 'Default Anonymous' to view its details.
-4.  Add the following scopes under the 'Scopes' section:
-
-        - schemas.https://huwelijksplanner.nl/schemas/hp.sdgProduct.schema.json.GET
-        - schemas.https://huwelijksplanner.nl/schemas/hp.availability.schema.json.GET
-
-Once you have completed these steps, you should have the Utrecht Huwelijksplanner project running on your local development environment with the necessary scopes configured for the anonymous user.
+With these steps completed, the frontend setup for the Huwelijksplanner project should be ready to use. If you encounter any issues during the installation process, seek assistance from the development team. Happy coding!
+
+## Admin UI - Setup Instructions
+
+Once the backend (and frontend) is up and running, the HuwelijksplannerBundle can be configured. To ensure proper functionality, the sources and Security Group (Default Anonymous user) need to be modified. Other adjustments are optional.
+
+### Configuration Steps:
+
+1.  **Users**
+    *   Change the passwords of the users if necessary.
+        *   Go to `Settings` in the Admin UI.
+        *   Navigate to the `Users` tab.
+        *   Select the user and edit the password.
```

Comment:
Maybe also recommend that the username (e-mail) is changed too.

Reply:
For the admin user that is, for the test users from the bundle we keep the default user names.